production/helm/loki/templates/_helpers.tpl

```diff
@@ -698,7 +639,7 @@ Create the service endpoint including port for MinIO.
 
 {{/* Determine if deployment is using object storage */}}
 {{- define "loki.isUsingObjectStorage" -}}
-{{- or (eq .Values.loki.storage.type "gcs") (eq .Values.loki.storage.type "s3") (eq .Values.loki.storage.type "azure") (eq .Values.loki.storage.type "swift") (eq .Values.loki.storage.type "alibabacloud") -}}
+{{- or (eq .Values.loki.storage.type "gcs") (eq .Values.loki.storage.type "s3") (eq .Values.loki.storage.type "azure") (eq .Values.loki.storage.type "swift") (eq .Values.loki.storage.type "alibabacloud") (eq .Values.loki.storage.type "cos") (eq .Values.loki.storage.type "bos") -}}
```

Comment:
Did you consider creating a list of the object stores and use the has function instead https://helm.sh/docs/chart_template_guide/function_list/#has-musthas to improve readability?

Reply:
Done